corio.py

```diff
@@ -301,6 +311,14 @@ def main(options):
                       'endpoint_url': S3_CFG.endpoint,
                       'use_ssl': S3_CFG["use_ssl"],
                       'seed': options.seed}
+    tests_details = dict()
+    tests_to_execute = dict()
+    jira_obj = None
+    jira_flg = options.test_plan
+    if jira_flg:
+        jira_id, jira_password = JiraApp.get_jira_credential()
+        jira_obj = JiraApp(jira_id, jira_password)
```

Comment:
Please move `        jira_id, jira_password = JiraApp.get_jira_credential()` inside the `__init__` of `JiraApp` , since `jira_id` and `jira_password` only being used inside JiraApp, will reduce complexity in main

Reply:
done